models/wrf/tutorial/README.rst

```diff
@@ -557,6 +559,11 @@ and look near the end. You will need to modify the lines staring with
 also want to modify this script to test running a single member first —
 just in case you have some debugging to do.
 
+However, be warned that to successfully complete the tutorial, including
+running the *driver.csh* script in Step 5, using a smaller ensemble 
+(e.g. < 20 members) can lead to spurious updates during the analysis step,
+causing the WRF simulation to fail. 
+
```

Comment:
The tutorial can run OK with as little as 20 ensemble members.  Wanted to avoid confusion with new users that using less than recommended 50 members, although good for quick troubleshooting, introduces problems if not done carefully.